### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by Dylanben144**
Fixes a basic UI problem and page view cron job

### CODE REVIEW

File: ckanext/privatedatasets/plugin.py [hunk 1]
```diff
@@ -341,18 +341,22 @@ def before_view(self, pkg_dict):
...
+                    'user_obj': tk.c.userobj
+                }
+
+                try:
+                    tk.check_access('resource_show', context, resource)
+                except tk.NotAuthorized:
+                    pkg_dict['resources'].remove(resource)
+                    pkg_dict = self.before_view(pkg_dict)
+            except:
+               print ("problem fixed")
```
Comment (D-Simona-G) [inline ckanext/privatedatasets/plugin.py]: Do we need this print here?

Comment (Dylanben144) [reply ckanext/privatedatasets/plugin.py]: We need something to fill the except and state the problem has occurred (Will help with debugging future occurrence of this problem). I will change to a better print message

File: ckanext/privatedatasets/plugin.py [hunk 2]
```diff
@@ -386,18 +390,22 @@ def before_delete(self, context, resource, resources):
...
+                'session': model.Session,
+                'user': tk.c.user,
+                'user_obj': tk.c.userobj
+            }
+            try:
+                tk.check_access('resource_show', context, resource_dict)
+            except tk.NotAuthorized:
+                resource_dict.clear()
+        except:
+            print ("problem fixed")
```
Comment (D-Simona-G) [inline ckanext/privatedatasets/plugin.py]: Same thing here, sorry I haven't noticed it the first time around

**Review by D-Simona-G — APPROVED** [review]

File: ckanext/privatedatasets/plugin.py (hunk 2 above)
Comment (Dylanben144) [reply ckanext/privatedatasets/plugin.py]: Added same fix

**Pull request merged**